site/src/pages/docs/advanced-spring-boot-integration.mdx

```diff
@@ -0,0 +1,420 @@
+# Spring Boot integration
+
+The [Spring](https://spring.io/) framework provides powerful features necessary for building web applications
+such as dependency injection, data binding, AOP and transaction management.
+Armeria is a lightweight microservices framework that offers capabilities comparable to existing full-stack web frameworks.
+It has rich support for [Apache Thrift](https://thrift.apache.org/) and [gRPC](https://grpc.io/).
+It also includes a user-friendly [web console](https://armeria.dev/docs/server-docservice)
+that allows you to send Thrift and gRPC requests directly from a web browser. Armeria supports both HTTP/1 and HTTP/2,
+and these protocols both support [cleartext](https://simple.wikipedia.org/wiki/Cleartext) and TLS cryptographic protocols.
+Armeria utilizes an [event loop](https://en.wikipedia.org/wiki/Event_loop) based on [Netty](https://netty.io/index.html),
+enabling non-blocking I/O operations. As a result, it demonstrates better performance in certain scenarios such as streaming media or
```

Comment:
This description might be useful if this page is published as an independent article of a blog. However, this is a part of Armeria documentation. I don't see we need to repeat Armeria features on this page.

What do you think of dropping from L5 to L11?
```
The [Spring](https://spring.io/) framework provides powerful features necessary for building web applications
such as dependency injection, data binding, AOP and transaction management.
By integrating your Spring application with Armeria, you can serve both legacy ...
```